src/metabase/api/dashboard.clj

```diff
@@ -716,25 +719,15 @@
                  field-id          (param-key->field-ids dashboard param-key)]
              [field-id value])))
 
-(s/defn chain-filter
-  "C H A I N filters!
-
-    ;; show me categories
-    (chain-filter 62 \"ee876336\" {})
-    ;; -> {:values          (\"African\" \"American\" \"Artisan\" ...)
-           :has_more_values false}
-
-    ;; show me categories that have expensive restaurants
-    (chain-filter 62 \"ee876336\" {\"6f10a41f\" 4})
-    ;; -> {:values          (\"Japanese\" \"Steakhouse\")
-           :has_more_values false}"
+(mu/defn chain-filter
+  "C H A I N filters!"
```

Comment:
The existing docstring has examples of how to use this and I think it's quite nice. Any reasons why you removed it?

Reply:
Yeah, the docstring is actually describing another `chain-filter` function, and doesn't work with this function. I'll move it to where it belongs, tho.

Reply:
Looking at the args you can see that this function expects a map first, not an integer, as in the docstrings.

Reply:
I think it'd still be worth having _some_ docstring for this function as well, at least with a description of the behavior if not also an example

Reply:
Looking through the git history, this docstring has been wrong since it's inception. I'll add something that's more correct.

Remember it's defined via mu/defn, so we get the arg info in the docstring too:

```
metabase.api.dashboard/chain-filter
 [dashboard param-key constraint-param-key->value]
 [dashboard param-key constraint-param-key->value query]
  Inputs: ([dashboard param-key constraint-param-key->value]
           [dashboard :- ms/Map param-key :- ms/NonBlankString constraint-param-key->value :- ms/Map query :- [:maybe ms/NonBlankString]])
  Return: :any
          

  C H A I N filters!

  Used to query for values that populate chained filter dropdowns and text search boxes.
  ```